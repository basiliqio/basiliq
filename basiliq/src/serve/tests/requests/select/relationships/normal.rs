use super::*;

crate::run_test_request!(
    get_multi,
    Method::GET,
    format!(
        "/public__peoples/{}/relationships/public__articles",
        PEOPLES_IDS[0]
    ),
    200
);

crate::run_test_request!(
    get_single,
    Method::GET,
    format!(
        "/public__peoples/{}/relationships/public__favorite_color",
        PEOPLES_IDS[0]
    ),
    200
);

crate::run_test_request!(
    get_unknown,
    Method::GET,
    format!(
        "/public__favorite_color/{}/relationships/public__articles",
        FAVORITE_COLOR_IDS[0]
    ),
    400
);
