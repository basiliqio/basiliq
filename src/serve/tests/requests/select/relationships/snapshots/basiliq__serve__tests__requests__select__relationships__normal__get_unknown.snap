---
source: basiliq/src/serve/tests/requests/select/relationships/normal.rs
expression: res

---
{
  "errors": {
    "id": "E20002",
    "status": 400,
    "title": "CibouletteUnknownRelationship",
    "detail": "No relationship `public__articles` for type `public__favorite_color`."
  }
}
